src/pages/api/callback.js

```diff
@@ -0,0 +1,10 @@
+import auth0 from "@/auth/auth0";
+
+export default async function callback(req, res) {
```

Comment:
maybe should have more specific name referencing auth? (or could it be used for other purposes?)